Makefile

```diff
@@ -20,8 +20,9 @@ CXXFLAGS := -Wall \
 	-Wmissing-include-dirs \
 	-Wno-unused-parameter \
 	-Wuninitialized \
-	-Wno-reorder
+	-Wno-reorder \
 
+LDFLAGS := -pthread -lGL -lGLU -lX11 -lXext -lm -lXxf86vm -lXrandr -lXi
```

Comment:
The only libraries I'm explicitly using are `-lGL -lm`. The rest might be necessary with your system's setup, but I'm not even using X11 at all. Maybe that's needed by GLFW on Linux, in which case I'll probably just extract the linker flags needed using pkgconfig.